feat: Windows.Devices.Sensors.Pedometer (#2070)
unoplatform/uno: doc/articles/features/windows-devices-sensors.md

`ReportInterval` property on WASM is currently not supported directly. Uno uses an approximation in the form of raising the `ReadingChanged` event only when enough time has passed since the last report. The event is raised a bit more often to make sure the gap caused by the filter is not too large, but this is in-line with the behavior of UWP `Magnetometer`.

`DirectionalAccuracy` is not reported on iOS, so it will always return `Unknown`.

## `Pedometer`

### Implementation notes

#### Android

> On Android, the first reading returns the cumulative number of steps since the device was first booted up. I have noticed the sensor does not correctly respect the requested reporting interval, so we implemented this manually to make sure the `ReadingChanged` events are triggered only after the `ReportInterval` elapses.

Reviewer: ```suggestion
On Android, the first reading returns the cumulative number of steps since the device was first booted up. The sensor may not correctly respect the requested reporting interval, so the implementation does this manually to make sure the `ReadingChanged` events are triggered only after the `ReportInterval` elapses.
```
